### ISSUE TITLE
Can I add more than 3 gateway in it?

### ISSUE THREAD

**Issue by mightChamp** [jarkko-hautakorpi/iBeacon-indoor-positioning-demo#6]
Want to implement 5 gateway to receive and locate ibeacons?
Can i use more gateway rather than 3?

**Comment by jarkko-hautakorpi** [comment]
You can use infinite number of stations/gateways. Stations are "dummy" in that they just report what they see. It is the React application which only pics 3 strongest stations for each beacon reported, and then calculates position between these. Other 2 weak measurements are ignored if beacon is visible in all 5 stations.

**Comment by mightChamp** [comment]
Thanks for your reply, I have done this by some modification in code.
Still I am not getting perfect result of location due to placing of gateway, I am using nrf52832 as gateway and send its data to dashboard through mqtt. Can you suggest more rssi filter algorithms. I added Kalman and average filter to get results. Results are considerable good if all gateway are in open space without wall. If wall comes in between beacon and Gateway, then result is not good show wrong placing. So if you can help me regarding it, then please help me.
So two point:

1) Location of gateway to get more accurate results?
2) overcome the wall effect of beacon?

**Comment by jarkko-hautakorpi** [comment]
Is it possible to do "RSSI mapping" of the space? Make a grid map and then measure RSSI in each spot, giving extra coefficient to certain stations when inside the troublesome triangle... One could allso use more than 3 stations to identify such location/coefficient.

**Comment by mightChamp** [comment]
how to properly define widthMeter in App.js?
RSSI mapping of space is possible, I will try and let you know.

**Comment by mightChamp** [comment]
how to properly define widthMeter in App.js?

**Comment by Alex-Telonic** [comment]
Yes, for some reason the widthMeter is hardcoded in the code. Should you not tell in the blog, that its value needs to changed according to the individual floorplan? (https://blog.shinovent.fi/2018/03/ibeacon-indoor-positioning-with-esp32.html)
In that case, you need to find out the complete meter width of your individual floorplan image.

**Comment by jarkko-hautakorpi** [comment]
> hardcoded in the code

True this needs to be fixed. Must forgot it in there when testing..

**Comment by mightChamp** [comment]
Ya true have to work on it.
